src/ui/ui.ts

```diff
@@ -560,4 +561,20 @@ export default class UI extends Phaser.GameObjects.Container {
   public getModeChain(): Mode[] {
     return this.modeChain;
   }
+
+  /**
+   * getGamepadType - returns the type of gamepad being used
+   * inputMethod could be "keyboard" or "touch" or "gamepad"
+   * if inputMethod is "keyboard" or "touch", then the inputMethod is returned
+   * if inputMethod is "gamepad", then the gamepad type is returned it could be "xbox" or "duelshock"
```

Comment:
isn't it supposed to be `dualshock`?
```suggestion
   * if inputMethod is "gamepad", then the gamepad type is returned it could be "xbox" or "dualshock"
```

Reply:
you are right!